syncd/NotificationProcessor.cpp

```diff
@@ -160,7 +160,7 @@ void NotificationProcessor::redisPutFdbEntryToAsicView(
         return;
     }
 
-    if (fdb->event_type == SAI_FDB_EVENT_LEARNED)
+    if (fdb->event_type == SAI_FDB_EVENT_LEARNED || fdb->event_type == SAI_FDB_EVENT_MOVE)
```

Comment:
this is assuming that fdb_entry didn't changed, just SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID attribute changed, so it went from 1 port to another port, so bridge_id or vlan_id stayed the same?

Reply:
Yes, I think it is the meaning of SAI_FDB_EVENT_MOVE, right?

Reply:
please check the tests, since i see that some assertions failed